src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,98 @@
+package duke;
+import java.util.List;
+
+
+/**
+ * Encapsulates a deadline Class
+ */
+public class TaskList {
+    private List<Task> taskList;
+
+    /**
+     * Creates a new taskList
+     * @param taskList The List of Tasks to be stored
+     */
+    public TaskList(List<Task> taskList) {
+        this.taskList = taskList;
+    }
+
+    /**
+     * Returns the List of Tasks that this class encapsulates
+     * @return the List of Tasks
+     */
+    public List<Task> getTaskList(){
+        return taskList;
+    }
+
+    /**
+     * Prints the list of Tasks
+     */
+    public String printTaskList(){
+        String returnString = "";
+        for(int i = 0; i<taskList.size(); i++){
+            Integer index = i+1;
+            returnString += (index + ". " + taskList.get(i));
+        }
+        return returnString;
+    }
+
+    /**
+     * Returns the number of tasks in the list
+     * @return the size of the List of Tasks
+     */
+    public int size(){
```

Comment:
What do you think of `getSize` instead of `size`? It might be clearer to even more unfamiliar readers